### PR TITLE
Increase brightness of android checkboxes with custom theme file

### PR DESCRIPTION
**Pull request by ryandoherty**

### CODE REVIEW

File: build_release_apk.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 rm -rf bin
+cp install/defaulttheme-0.png .buildozer/android/platform/python-for-android/dist/racecapture/private/lib/python2.7/site-packages/kivy/data/images/defaulttheme-0.png
```
Comment (brentpicasso) [inline build_release_apk.sh]: wow, right into the .buildozer folder. Seems extreme but awesome. Is this primarily b/c they don't have a good way to do this officially?

**Comment by brentpicasso** [comment]
Think we need to worry about this for windows?

File: build_release_apk.sh (hunk 1 above)
Comment (ryandoherty) [inline build_release_apk.sh]: Yeah, I had tried to do it the (semi) 'official' way in Kivy by calling some kivy functions to load a different theme file, but it never worked and there's little to no documentation on how custom theming works. This is relatively safe b/c if for some reason the file name changes with a Kivy upgrade, you just get the Kivy theme instead.

**Comment by ryandoherty** [comment]
Windows seems to be ok with the default theme because of either higher contrast monitors or their color profiles.

**Comment by brentpicasso** [comment]
Great! if tested and looks good on android then update changelog and then :shipit: Also, maybe update version in main.py to 1.3.10 as well.

**Comment by ryandoherty** [comment]
Yup, tested on android and OSX to verify changes didn't break the OSX build/theme. Will update changelog.

**Pull request merged**